docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -3350,6 +3402,68 @@ components:
       type: object
     SqlParameterEditor:
       type: object
+    StatisticDto:
+      required:
+      - name
+      - value
+      type: object
+      properties:
+        name:
+          $ref: '#/components/schemas/StatisticName'
+        value:
+          type: integer
+          format: int64
+    StatisticName:
+      type: string
+      enum:
```

Comment:
I'm wondering, if for readability's sake, shouldn't add an event type prefix, in all names, 

`SELECT_SCENARIO_FILTER`
`CLICK_EDIT_UNDO`
`CLICK_TEST_GENERATED` -> because someone can open the modal, but for some reason doesn't generate the file. The other thing is a statistic, how many user opens the generate file dialog and don't generate them?

Also, if we have `SCENARIO_FILTER` which is about selecting the filter, does it make sense to send an unselected filter statistic?

Reply:
Yes, I agree. I will change it